packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -11,8 +11,8 @@ dependencies: {{#enable_dev_menu}}
   alice: ^1.0.0-dev.12
   alice_dio: ^1.0.6{{/enable_dev_menu}}
   collection: ^1.18.0
-  copy_with_extension: ^5.0.4{{#enable_pin_code}}
-  crypto: ^3.0.6{{/enable_pin_code}}
+  copy_with_extension: ^5.0.4{{#has_authentication}}
```

Comment:
I believe copy_with_extension version needs to be set to 6.0.0 once we release the widget_toolkit package.

FYI: @JvnSlv

Reply:
Updated.